dle/cfn/deploy_ec2.yaml

```diff
@@ -0,0 +1,151 @@
+    Description: DB Endpoint
+  AccessKeyParameter:
+    Type: String
+    Default: ""
+    Description: DB Endpoint
+  SecretAccessKeyParameter:
+    Type: String
+    Default: ""
+    Description: DB Endpoint
+
+Resources:
+  globalRxKey:
+    Type: AWS::EC2::KeyPair
+    Properties: 
+      KeyName: !Ref KeyIDParameter
+      PublicKeyMaterial: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDel37v9xkR2BF6xyaXvSylZ7ZL5nUDynVaPPxXt7JTPVj3AMp6BWx/+YJh75lorOx+rsJJo8uwq67bnfV/BBPkRyvXDU68U2BSmg7YOxDYJH4d6tizQV3NKD0v3NSYdQYk11GJx1HOe/5Ye7YujTaF7bwoDXu45vrE2HzMZLTdDveeMl5866Q78GMOZLTvw9okjT1d/THHmXEwzAftnwir1PuEtEvrQ6QZ0iVO0hDAbSbXYeZl1g/G1qi+Z0AI+aeyshZQFzhYdWA2M5tSNe71RlVw6reBdQ+mP4vQyPI72GxWpTU8mQ65PoB/EEayaF+bQUewaC7lQ1LyygVZbS68KFbUjD03y5DMcdyWXRIjTYGaB0oB1zzQ9A6eGuEmPw4lFTkKRllfRXmAj81E9lp0Ge+eSKmRuuahckvFtDnFD2tSVvVSXnKg/WivNDyVOYVzTPEePdnCFw4EYcwB1anKnBX/0RbfBzq3Cx7S3fVYcJIEwJ66jVKeuQEpHzO9qz0=
+      Tags:
+        - Key: Name
+          Value: globalRx1
+
+  globalRxEc2:
+    Type: AWS::EC2::Instance
+    DependsOn: globalRxKey
+    Properties:
+      ImageId: !Ref ImageIDParameter
+      InstanceType: !Ref InstanceTypeParameter
+      IamInstanceProfile: "globalRX"
+      KeyName: !Ref KeyIDParameter
+      BlockDeviceMappings: 
+        - DeviceName: "/dev/xvda"
+          Ebs: 
+            DeleteOnTermination: True
+            VolumeSize: 100
+            VolumeType: gp3
+      SecurityGroupIds:
+        - !GetAtt "globalRxSecGroup.GroupId"
+      SubnetId: "subnet-01f4190f"
+      Tags:
+        - Key: Name
+          Value: !Ref NameParameter
+      UserData:
+        Fn::Base64: 
+          !Sub |
+            #!/bin/bash
+            yum update -y
+            yum install docker -y
+            systemctl enable docker
+            systemctl start docker
+            yum install git -y
+            curl -L https://github.com/docker/compose/releases/latest/download/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+            chmod +x /usr/local/bin/docker-compose
+
+            cd /home/ec2-user
+            git clone https://github.com/globalrx/globalrx.git
+            cd /home/ec2-user/globalrx/dle
+            git checkout master
+
+
+            export AWS_ACCESS_KEY=${AccessKeyParameter}
+            export AWS_SECRET_KEY=${SecretAccessKeyParameter}
+            
+            cd /home/ec2-user/globalrx/dle
+            IP=$(dig +short txt ch whoami.cloudflare @1.0.0.1  | tr -d '"')
+            aws cloudformation create-stack --stack-name myteststack --capabilities CAPABILITY_IAM --template-body file://cfn/deploy_django.yml --parameters ElasticIPAddress=$IP
+            docker-compose -f elastic_compose.yml up
+
+
+
+  globalRxSecGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Open 22 and 8000
+      VpcId: !Ref VPCParameter
+      SecurityGroupIngress:
+          - IpProtocol: tcp
+            FromPort: '22'
+            ToPort: '22'
+            CidrIp: 0.0.0.0/0 
```

Comment:
Ideally should limit this to just IPs that need to SSH. And then we can deal with some drift as people add their IPs in Console, or add another security group just to whitelist IPs. If we don't have keys on it IDK if it matters